Thanks @cyrush !

In the failing CI runs it looks like the HDF5 is correctly building with MPI/parallel, but the CGNS cmake is showing
```
 CMake Error at CMakeLists.txt:429 (message):
  HDF5 has been found, but is missing parallel support.
```

@brtnfld anything obviously wrong with the build there?

--- a/.github/workflows/build_conduit_gcc.yml
+++ b/.github/workflows/build_conduit_gcc.yml
@@ -7,7 +7,7 @@ on:
 jobs:
   build_basic:
     name: Ubuntu Build Conduit GCC
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     env:
       CC: gcc
       CXX: g++
@@ -16,7 +16,8 @@ jobs:
     - name: Install System Deps
       run: |
            sudo apt-get update
-           sudo apt-get install binutils \
+           sudo env DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
+           sudo apt-get install -y binutils \
                                 gcc \
                                 g++ \
                                 gfortran \
@@ -78,7 +79,6 @@ jobs:
         cmake --install build
     - name: Check Install
       run: |
-        echo "**** Checking Conduit using-with-cmake-mpi example"
         export CC=/usr/bin/mpicc
         export CXX=/usr/bin/mpicxx
         cd install/examples/conduit/using-with-cmake-mpi
@@ -87,7 +87,7 @@ jobs:
         mpiexec -n 2 ./build/conduit_mpi_example
   build_basic_pyvenv:
     name: Ubuntu Build Conduit GCC with Python VENV
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     env:
       CC: gcc
       CXX: g++
@@ -96,7 +96,8 @@ jobs:
     - name: Install System Deps
       run: |
            sudo apt-get update
-           sudo apt-get install binutils \
+           sudo env DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
+           sudo apt-get install -y binutils \
                                 gcc \
                                 g++ \
                                 gfortran \

--- a/scripts/build_conduit/2026_01_06_cgns_hdf5_2.patch
+++ b/scripts/build_conduit/2026_01_06_cgns_hdf5_2.patch
@@ -1,0 +1,28 @@
+From 2f97eded4c027bf2514828abf9f0a1a05ae2b9dc Mon Sep 17 00:00:00 2001
+From: Cyrus Harrison <cyrush@llnl.gov>
+Date: Tue, 6 Jan 2026 12:28:44 -0800
+Subject: [PATCH] patch
+
+---
+ CMakeLists.txt | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8a400b8..d89596c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -425,7 +425,10 @@ if (CGNS_ENABLE_HDF5 AND HDF5_NEED_MPI)
+   set(CGNS_ENABLE_PARALLEL "OFF" CACHE BOOL "Enable or disable parallel interface ?")
+   mark_as_advanced(CLEAR CGNS_ENABLE_PARALLEL)
+   # Check that HDF5 has parallel support
+-  if (NOT (HDF5_IS_PARALLEL OR HDF5_ENABLE_PARALLEL))
++  # HDF5 2.0 changed the cmake var that indicates MPI support to HDF5_PROVIDES_PARALLEL, see:
++  # https://www.hdfgroup.org/2025/11/10/release-of-hdf5-2-0-0-newsletter-207/
++  # https://github.com/HDFGroup/hdf5/pull/5716
++  if (NOT (HDF5_IS_PARALLEL OR HDF5_ENABLE_PARALLEL OR HDF5_PROVIDES_PARALLEL))
+     message(FATAL_ERROR "HDF5 has been found, but is missing parallel support.")
+   endif()
+ else ()
+-- 
+2.50.1 (Apple Git-155)
+

--- a/scripts/build_conduit/build_conduit.sh
+++ b/scripts/build_conduit/build_conduit.sh
@@ -270,7 +270,7 @@ if ${build_cgns}; then
 if [ ! -d ${cgns_src_dir} ]; then
   echo "**** Downloading ${cgns_tarball}"
   curl -L https://github.com/CGNS/CGNS/archive/refs/tags/v${cgns_version}.tar.gz -o ${cgns_tarball}
-  tar -xzf ${cgns_tarball}
+  tar ${tar_extra_args} -xzf ${cgns_tarball} -C ${source_dir}
 
   # hdf5 2.0 patch
   cd  ${cgns_src_dir}

--- a/scripts/build_conduit/build_conduit.sh
+++ b/scripts/build_conduit/build_conduit.sh
@@ -259,10 +259,10 @@ fi # build_hdf5
 ################
 
 cgns_version=4.5.0
-cgns_tarball=cgns-${cgns_version}.tar.gz
+cgns_src_dir=$(ospath ${source_dir}/CGNS-${cgns_version})
+cgns_build_dir=$(ospath ${build_dir}/cgns-${cgns_version}/)
 cgns_install_dir=$(ospath ${install_dir}/cgns-${cgns_version}/)
-cgns_src_dir=$(ospath ${root_dir}/CGNS-${cgns_version})
-cgns_build_dir=$(ospath ${root_dir}/build/cgns-${cgns_version}/)
+cgns_tarball=$(ospath ${source_dir}/cgns-${cgns_version}.tar.gz)
 
 # build only if install doesn't exist
 if [ ! -d ${cgns_install_dir} ]; then
@@ -271,6 +271,11 @@ if [ ! -d ${cgns_src_dir} ]; then
   echo "**** Downloading ${cgns_tarball}"
   curl -L https://github.com/CGNS/CGNS/archive/refs/tags/v${cgns_version}.tar.gz -o ${cgns_tarball}
   tar -xzf ${cgns_tarball}
+
+  # hdf5 2.0 patch
+  cd  ${cgns_src_dir}
+  patch -p1 < ${script_dir}/2026_01_06_cgns_hdf5_2.patch
+  cd ${root_dir}
 fi
 
 
@@ -399,7 +404,7 @@ if [ ! -d ${caliper_src_dir} ]; then
   echo "**** Downloading ${caliper_tarball}"
   curl -L https://github.com/LLNL/Caliper/archive/refs/tags/v${caliper_version}.tar.gz -o ${caliper_tarball}
   tar ${tar_extra_args} -xzf ${caliper_tarball} -C ${source_dir}
-  # windows specifc patch
+  # windows specific patch
   cd  ${caliper_src_dir}
   if [[ "$build_windows" == "ON" ]]; then
     patch -p1 < ${script_dir}/2024_08_01_caliper-win-smaller-opts.patch

